packages/theme/css/characteristics/overlayable.css

```diff
@@ -1,7 +1,7 @@
 .uitk-light,
 .uitk-dark {
   --uitk-overlayable-shadow-scroll: var(--uitk-shadow-1);
-  --uitk-overlayable-shadow-border-region: var(--uitk-shadow-2);
+  --uitk-overlayable-shadow-borderRegion: var(--uitk-shadow-2);
```

Comment:
hmmm, is this valid?

Reply:
this is how it is in Figma

Reply:
typo?

Reply:
this came from design, we together decided to make any subitems/concepts camel case for easier reading and separation of token part semantics (see e.g. size foundation for examples)